tests/pawn-shop.ts

```diff
@@ -159,25 +162,86 @@ describe("PawnHub", () => {
       LENDER_KEYPAIR.publicKey
     );
     await pawnMint.mintTo(borrowerPawnTokenAccount, BORROWER_KEYPAIR, [], 1);
+
+    const tx = new Transaction();
+    // those address derivation are no longer in the package, why?
+    const [metadata] = await PublicKey.findProgramAddress(
+      [
+        Buffer.from("metadata"),
+        METAPLEX_PROGRAM_ID.toBuffer(),
+        pawnMint.publicKey.toBuffer(),
+      ],
+      METAPLEX_PROGRAM_ID
+    );
+    tx.instructions.push(
+      createCreateMetadataAccountV2Instruction(
+        {
+          metadata,
+          mint: pawnMint.publicKey,
+          mintAuthority: BORROWER_KEYPAIR.publicKey,
+          payer: BORROWER_KEYPAIR.publicKey,
+          updateAuthority: BORROWER_KEYPAIR.publicKey,
+        },
+        {
+          createMetadataAccountArgsV2: {
+            data: {
+              name: "Test",
+              symbol: "TST",
+              uri: "https://google.com",
+              sellerFeeBasisPoints: 0,
+              creators: null,
+              collection: null,
+              uses: null,
+            },
+            isMutable: true,
+            //collectionDetails: null,
+          },
+        }
+      )
+    );
+    const [edition] = await PublicKey.findProgramAddress(
+      [
+        Buffer.from("metadata"),
+        METAPLEX_PROGRAM_ID.toBuffer(),
+        pawnMint.publicKey.toBuffer(),
+        Buffer.from("edition"),
+      ],
+      METAPLEX_PROGRAM_ID
+    );
+    tx.instructions.push(
+      createCreateMasterEditionV3Instruction(
```

Comment:
what's the reason behind making it a master's edition?

Reply:
https://docs.metaplex.com/about/terminology#normal-nft
Once the master edition is created the token control is surrendered to the metaplex program, only then it becomes a "Solana NFT".

For instance you can create a mint and metadata, then make 1, and later on mint another one.
If there is a master edition constraining the supply `maxSupply`, even the NFT update authority cannot do that.